src/components/Filter.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {FormControlLabel, Checkbox, Button, Drawer, Divider, IconButton} from '@mui/material';
+import {useState} from 'react';
+import {RangeSlider} from '../Helper_components/RangeSlider';
+
+
+
+
+const listOfGenres = [
+  'Comedy',
+  'Fantasy',
+  'Crime',
+  'Drama',
+  'Music',
+  'Adventure',
+  'History',
+  'Thriller',
+  'Animation',
+  'Family',
+  'Mystery',
+  'Biography',
+  'Action',
+  'Film-Noir',
+  'Romance',
+  'Sci-Fi',
+  'War',
+  'Western',
+  'Horror',
+  'Musical',
+  'Sport',
+].sort();
+
+
+export const Filter = (props) => {
+
+  const [selectedGenres, setSelectedGenres] = useState([]);
+  const [range, setRange] = useState([1930, 2100]);
+  const [open, setOpen] = useState(false);
+  
+
+  const handleChange = (event) => {
+  if (selectedGenres.includes(event.target.value)){
+    setSelectedGenres(selectedGenres.filter((genre) => event.target.value !== genre))
+  } else {
+    setSelectedGenres([event.target.value, ...selectedGenres])
+  }
+};
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+  
+
+  const cleanFilter = () => {
+    props.handleClearFilter()
+  }
+
+  
+  return (
+    <div>
+      <div className='filter'>
+        <p>Filter Movies by Genre/Year:</p>
+        <Button sx={{backgroundColor: 'white', color: 'black'}} variant="contained" onClick={handleDrawerOpen}>Filter</Button>
```

Comment:
Let's move this to the appropriate css or let's use styled component.

Reply:
I managed to style it by using makeStyles function and adding a css class as below:

**const useStyles = makeStyles({
  button: {
    backgroundColor: 'white',
    color: 'black',
  },**
.
.
.
<Button **className={classes.button}** variant="contained" onClick={handleDrawerOpen}>Filter</Button>
<Button **className={classes.button}** variant='contained' onClick={cleanFilter}>Clear Filter</Button>

Reply:
I managed to style them by using makeStyles function as below:

![Screenshot_6](https://user-images.githubusercontent.com/70068774/174501567-a0f813ed-c320-4c4c-ab20-3a72861d6474.png)
![Screenshot_7](https://user-images.githubusercontent.com/70068774/174501570-974b5bbf-e309-4e81-b554-19d314e93ae6.png)